src/js/subtraction/querys.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from "react-query";
+import { Request } from "../app/request";
+
+function fetchSubtractions({ term, page }) {
+    return Request.get("/subtractions")
+        .query({ find: term, page })
+        .then(response => {
+            return response.body;
+        });
+}
+
+export function useFetchSubtractions(term: string, page: number) {
+    return useQuery("subtractions", () => fetchSubtractions({ term, page }));
+}
```

Comment:
Two things:

1. useQuery should be passed typing information
2. Take a look at some of the `QueryKeys` functions for how to define query keys